README.md

```diff
@@ -0,0 +1,38 @@
+# Roulette Telegram Chat
+
+![Man speaking with women](./docs/logo.png)
+The idea is to create a chat for communication between two random people without revealing their identities.
+
+## Use case diagram
+
+![Interaction between user, moderator and system](./docs/useCase.png)
+Main points:
+
+### User
+
+- can register in system(by pressing button start)
+- can change companion to new one
+- can stop communication
+- report companion abuse
+- block companion to not interact with him anymore
+- get companion "fake" identity (name, age, hobbies)
```

Comment:
how it will be implemented? Some dialog onboarding process?

Reply:
need more clarification

Reply:
add sentimental analysis

Reply:
for example chat with new user starts we can make some short intro with that fake info